konrad/radiation/rrtmg.py

```diff
@@ -63,18 +65,22 @@ def init_radiative_state(self, atmosphere, surface):
 
         climt.set_constants_from_dict({"stellar_irradiance": {
                 "value": self.solar_constant, "units": 'W m^-2'}})
+        if self._mcica:
```

Comment:
I know that this is not introduced in this PR, but it is good practice to start boolean variables with a prefix like ``is_`` (e.g. ``self._is_mcica``). This helps to determine the type of the variable at first glance (at least for me).